### ISSUE TITLE
Architectural PMU Version Info

### ISSUE THREAD

**Issue by blu** [hrw/arm-socs-table#57]
**Describe the system**
System name: any SoC with ID_AA64DFR0_EL1[11:8] PMUVer != 0
Vendor name: multiple
SoC name: multiple
URL to product page:
https://developer.arm.com/documentation/ddi0595/2021-06/AArch64-Registers/ID-AA64DFR0-EL1--AArch64-Debug-Feature-Register-0?lang=en

**Additional context**
The architectural (where present) PMU version is a useful implementation detail. It's not part of the cpuid, nor it depends on armv8-revision; instead it's fully described by a debug feature register -- ID_AA64DFR0_EL1[11:8] PMUVer; additionally ID_AA64DFR0_EL1[35:32] PMSVer is becoming increasingly useful. Reporting the numerical value of those DFR fields would be valuable.

Furthermore, it's worth considering reporting the architectural PMU events per the given implementation [1], but that's more involved as it requires a major SoC record overhaul.

[1] 

```
$ ls -l /sys/devices/armv8_pmuv3/events/
total 0
-r--r--r-- 1 root root 4096 ное 22 16:41 br_mis_pred
-r--r--r-- 1 root root 4096 ное 22 16:41 br_mis_pred_retired
-r--r--r-- 1 root root 4096 ное 22 16:41 br_pred
-r--r--r-- 1 root root 4096 ное 22 16:41 br_retired
-r--r--r-- 1 root root 4096 ное 22 16:41 cid_write_retired
-r--r--r-- 1 root root 4096 ное 22 16:41 cpu_cycles
-r--r--r-- 1 root root 4096 ное 22 16:41 exc_return
-r--r--r-- 1 root root 4096 ное 22 16:41 exc_taken
-r--r--r-- 1 root root 4096 ное 22 16:41 inst_retired
-r--r--r-- 1 root root 4096 ное 22 16:41 l1d_cache
-r--r--r-- 1 root root 4096 ное 22 16:41 l1d_cache_refill
-r--r--r-- 1 root root 4096 ное 22 16:41 l1d_cache_wb
-r--r--r-- 1 root root 4096 ное 22 16:41 l1d_tlb_refill
-r--r--r-- 1 root root 4096 ное 22 16:41 l1i_cache
-r--r--r-- 1 root root 4096 ное 22 16:41 l1i_cache_refill
-r--r--r-- 1 root root 4096 ное 22 16:41 l1i_tlb_refill
-r--r--r-- 1 root root 4096 ное 22 16:41 mem_access
-r--r--r-- 1 root root 4096 ное 22 16:41 memory_error
-r--r--r-- 1 root root 4096 ное 22 16:41 stall_backend
-r--r--r-- 1 root root 4096 ное 22 16:41 stall_frontend
-r--r--r-- 1 root root 4096 ное 22 16:41 sw_incr
-r--r--r-- 1 root root 4096 ное 22 16:41 ttbr_write_retired
```

**Comment by hrw** [comment]
Would be nice to have POSIX sh script to gather such information. So it can be run even on Android devices with their limited environment.
